traffic_ops/app/lib/API/Cachegroup.pm

```diff
@@ -164,6 +166,7 @@ sub update {
 		latitude                       => $params->{latitude},
 		longitude                      => $params->{longitude},
 		parent_cachegroup_id           => $params->{parentCachegroupId},
+		fallback_to_closest            => $params->{fallbackToClosest},
```

Comment:
Because this field was not added to Traffic Portal, users of the TP will always be setting this to null when they update a cache group. Is that OK? 

I think you need to add this field to the Traffic Portal - form.cacheGroup.tpl.html